workshop/7-PerformanceOptimization/7.1-Layer0.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
it should be numbered 8 for the workshop
(7 is location services)

Reply:
Changed.